### ISSUE TITLE
JavaScript errors sporadically when downloading videos

### ISSUE THREAD

**Issue by OwenTheBowen** [StefanLobbenmeier/youtube-dl-gui#104]
Whenever I download videos from a playlist, I sporadically get the following error popping up in a window:

---

**A JavaScript error occurred in the main process**
```
Uncaught Exception:
TypeError: Cannot use 'in' operator to search for 'fragment_count' in 4
	at
C:\Users\*user*\AppData\Local\Programs\youtube-dl-gui\resources\app.asar\mo...:38
	at Socket.<anonymous>
(C:\Users\*user*\AppData\Local\Programs\youtube-dl-gui\resources\app.asar\m...:25)
	at Socket.emit (node:events:513:28)
	at addChunk (node:internal/streams/readable:324:12)
	at readableAddChunk (node:internal/streams/readable:293:11)
	at Socket.push (node:internal/streams/readable:234:10)
	at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)
```

---

If it perhaps helps diagnose the issue, I have killed the process once in task manager.
Additionally, the video download pauses when the error pops up, but once I press "Ok" it starts downloading the next video in the playlist.

**Comment by OwenTheBowen** [comment]
Side note:
I reinstalled the software once and the errors persist.

**Comment by StefanLobbenmeier** [comment]
Hmm that is a a really weird error. Not sure what could cause it, from the stacktrace it is something internal to node. Might be something interfering with the connection between the electron frontend and backend, but no idea sorry

**Comment by OwenTheBowen** [comment]
Alright. In that case perhaps I could try a more complete reinstall. I'm assuming uninstalling the app would still leave some files around the place? What are these files so I can purge them.

**Comment by OwenTheBowen** [comment]
I uninstalled and then re-installed from the installer I have sitting in my software archive (Version 2.4.10) and it works fine. Seemingly the issue is only present in the most recent version, as I also didn't have it before I updated to that from the one before.

I'll keep downloading (I have a long playlist to download) and let you know if it's run without errors once it's done. It's about half complete right now.
